statusHandler/prometheusHandler.go

```diff
@@ -0,0 +1,105 @@
+package statusHandler
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/prometheus/client_golang/prometheus"
+	prometheusUtils "github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+// PrometheusStatusHandler will define the handler which will update prometheus metrics
+type PrometheusStatusHandler struct {
```

Comment:
Should find a way how to test this class.

Reply:
added some tests